source/documentation/information/githubfeature.html.md.erb

```diff
@@ -6,3 +6,43 @@ review_in: 6 months
 ---
 
 # GitHub Feature Requests
+
+# Making Feature Requests to GitHub
```

Comment:
This is a h1 heading, same as line 8, should it be a h2 heading and the heading below it be h3?

Reply:
I've yet to look at the page layout fully - I'll be re-doing all the headings